api/meta_data.go

```diff
@@ -30,8 +31,12 @@ func (c *Client) SetMetaData(ctx context.Context, jobId string, metaData *MetaDa
 }
 
 // Gets the meta data value
-func (c *Client) GetMetaData(ctx context.Context, jobId string, key string) (*MetaData, *Response, error) {
-	u := fmt.Sprintf("jobs/%s/data/get", jobId)
+func (c *Client) GetMetaData(ctx context.Context, scope, id, key string) (*MetaData, *Response, error) {
+	if scope != "job" && scope != "build" {
+		return nil, nil, errors.New("scope must either be job or build")
+	}
+
+	u := fmt.Sprintf("%ss/%s/data/get", scope, id)
```

Comment:
is the `%ss` here a mistake or is the path `jobs` and `builds`?

Reply:
the path is `jobs` and `builds` :)